docs/index.md

```diff
@@ -1,3 +1,172 @@
 # JobSphere
 
-One stop for all your interview, internship and Job Hunting needs in this technical world!
\ No newline at end of file
+This project aims to be the ultimate resource for students transitioning into the professional world. Here, you'll find everything you need to land that dream job, all in one convenient location.
+
+**Key Features:**
+
+* **Career Development:** Craft a stellar resume using our library of templates and guides. Find valuable internship opportunities to gain real-world experience.
+* **Tech Interview Prep:** Conquer your technical interviews with our extensive collection of coding challenges. Explore solutions and explanations to hone your problem-solving skills.
+* **Centralized Resource:** No more juggling through countless websites. We strive to be your one-stop shop for all career-related information and preparation.
+
+**Get Started Today!**
+
+This is just the beginning. We're constantly adding new resources and features to help you navigate the exciting yet challenging world of professional careers. Dive deeper into the following sections to explore what we offer:
+
+1. [Research Internships](#research-internships)
+2. [Career Guides](#career-guides)
+3. [Interview Preparation](#interview-preparation)
+
+## 1. Research Internships
+
+
+
+
+
+The following are some global research internship positions:
+
+**CANADA**
+
+
+  * [University of Alberta Research Experience (UARE)](https://www.ualberta.ca/admissions-programs/visiting-student-and-internship-programs/research-internships/ualberta-research-experience/index.html)
+
+**US**
+  * [LPI Summer Intern Program in Planetary Science](https://www.lpi.usra.edu/lpiintern/application/), for both national and international students.
```

Comment:
Recheck the formatting here, when served as a mkdocs site.

Reply:
Use a newline, by doing this the output looks like:

![image](https://github.com/acmbpdc/JobSphere/assets/62911996/81b2383e-7f3a-4c5a-9c91-c7999cc0ad28)

It does not group with the other bullet points